sklearn/compose/_column_transformer.py

```diff
@@ -372,6 +376,53 @@ def _fit_transform(self, X, y, func, fitted=False):
             else:
                 raise
 
+    def _calculate_inverse_indices(self, X):
+        """
+        Private function to calcuate indicies for inverse_transform
+        """
+        # checks for overlap
+        all_indexes = set()
```

Comment:
minor nit, but maybe also use 'indices' instead of 'indexes' since all other variables do that?